src/components/calendar/CalendarHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import { ChevronLeftIcon, ChevronRightIcon } from '@chakra-ui/icons';
+import { HStack, IconButton, Select } from '@chakra-ui/react';
+
+import { useCalendarView } from '../../hooks/useCalendarView';
+
+export function CalendarHeader() {
+  const { view, setView, navigate } = useCalendarView();
+
+  return (
+    <HStack mx="auto" justifyContent="space-between">
+      <IconButton
+        aria-label="Previous"
+        icon={<ChevronLeftIcon />}
+        onClick={() => navigate('prev')}
+      />
+      <Select
+        aria-label="view"
+        value={view}
+        onChange={(e) => setView(e.target.value as 'week' | 'month')}
+      >
+        <option value="week">Week</option>
+        <option value="month">Month</option>
+      </Select>
+      <IconButton aria-label="Next" icon={<ChevronRightIcon />} onClick={() => navigate('next')} />
```

Comment:
전반적인 코드에 관한 질문을 남겨주셔서

저는 이런 코드는
```suggestion
      <IconButton 
        aria-label="Next" 
        icon={<ChevronRightIcon />} 
        onClick={() => navigate('next')} 
       />
```

이렇게 작성하는 편입니다. 가로로 길게 눈알이 이동하지 않게 하는편이 좋은듯해요. 이거는 확실히 개인선호의 영역이니까 참고만해주세요.